source/common/config/rush/deploy.json

```diff
@@ -140,109 +140,5 @@
     //     // "@types/*"
     //   ]
     // }
-    {
-      "projectName": "@awssolutions/cdf-assetlibrary-client",
-      "dependenciesToExclude": ["aws-sdk"]
-    },
-    {
-      "projectName": "@awssolutions/cdf-assetlibraryhistory-client",
-      "dependenciesToExclude": ["aws-sdk"]
-    },
-    {
-      "projectName": "@awssolutions/cdf-commands-client",
-      "dependenciesToExclude": ["aws-sdk"]
-    },
-    {
-      "projectName": "@awssolutions/cdf-organizationmanager-client",
-      "dependenciesToExclude": ["aws-sdk"]
-    },
-    {
-      "projectName": "@awssolutions/cdf-notifications-client",
-      "dependenciesToExclude": ["aws-sdk"]
-    },
-    {
-      "projectName": "@awssolutions/cdf-provisioning-client",
-      "dependenciesToExclude": ["aws-sdk"]
-    },
-    {
-      "projectName": "@awssolutions/cdf-errors",
-      "dependenciesToExclude": ["aws-sdk"]
-    },
-    {
-      "projectName": "@awssolutions/cdf-express-middleware",
-      "dependenciesToExclude": ["aws-sdk"]
-    },
-    {
-      "projectName": "@awssolutions/cdf-lambda-invoke",
-      "dependenciesToExclude": ["aws-sdk"]
-    },
-    {
-      "projectName": "@awssolutions/cdf-logger",
-      "dependenciesToExclude": ["aws-sdk"]
-    },
-    {
-      "projectName": "@awssolutions/cdf-assetlibrary",
-      "dependenciesToExclude": ["aws-sdk"]
```

Comment:
for all of the service packages, this was previously a no-op as the `aws-sdk` code was bundled into the lambda zip via esbuild.